src/main/java/com/dev/cinema/controllers/HelloController.java

```diff
@@ -1,12 +1,74 @@
 package com.dev.cinema.controllers;
 
+import com.dev.cinema.exception.AuthenticationException;
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.model.Movie;
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.model.Order;
+import com.dev.cinema.model.ShoppingCart;
+import com.dev.cinema.model.User;
+import com.dev.cinema.service.AuthenticationService;
+import com.dev.cinema.service.CinemaHallService;
+import com.dev.cinema.service.MovieService;
+import com.dev.cinema.service.MovieSessionService;
+import com.dev.cinema.service.OrderService;
+import com.dev.cinema.service.ShoppingCartService;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class HelloController {
```

Comment:
```suggestion
public class InjectController {
```
оскільки логіка повністю змінилася, бажано і назву змінити